mk/travis.sh

```diff
@@ -64,17 +64,31 @@ rustc --version
 
 if [[ "$MODE_X" == "RELWITHDEBINFO" ]]; then mode=--release; fi
 
-CC=$CC_X CXX=$CXX_X cargo build -j2 ${mode-} --verbose --target=$TARGET_X
+if [[ "$KCOV" == "1" ]]; then
+  norun=--no-run
+  RUSTFLAGS_X="-C link-dead-code"
+  ./mk/travis-install-kcov.sh
+fi
```

Comment:
If we do this, then kcov will get built from source even before we've built *ring*. This would increase the latency on seeing *ring* build failures. Instead, let's tun traivs-install-kcov.sh only after `cargo test` passes.